manifests/postfix_exporter.pp

```diff
@@ -82,6 +82,7 @@
 
   # scrape job configuration
   Boolean        $export_scrape_job = false,
+  Optional[Stdlib::Host] $scrape_host     = undef,
```

Comment:
We don't need to align all the `=`, but also we don't need to add too much whitespace

```suggestion
  Optional[Stdlib::Host] $scrape_host  = undef,
```